src/shared/services/auth/auth.ts

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2018 - 2021 Gemeente Amsterdam
+import { UserCredentials } from 'containers/App/types'
+import configuration from '../configuration/configuration'
+import Keycloak from './services/keycloak'
+import ImplicitAuth from './services/implicit-auth'
+
+const useKeycloak =
```

Comment:
I choose a different name for this variable; in the React context, this name signifies a hook.